src/components/debug/debug.js

```diff
@@ -37,16 +37,17 @@ const Content = styled.div`
   display: ${({ open }) => (open ? "bloc" : "none")};
 `;
 
-const Debug = ({ async }) => {
+const Debug = ({ async, setAsync }) => {
```

Comment:
`async` is also a reserved keywords, it's weird in review mode 😛

Reply:
I see it now, suprisingly, no problem in runtime